ReSharper.FSharp/src/FSharp.Psi/src/Impl/Tree/TopBinding.cs

```diff
@@ -37,7 +37,9 @@ public TreeTextRange GetNameRange()
 
     public ITypeDeclaration GetContainingTypeDeclaration() => GetContainingNode<ITypeDeclaration>();
 
-    public XmlNode GetXMLDoc(bool inherit) => null;
+    public XmlNode GetXMLDoc(bool inherit) =>
+      FirstChild is XmlDocBlock xmlDocBlock ? xmlDocBlock.GetXML(null) : null;
```

Comment:
Do we ever get the xml from the binding node and not from its head pattern declared element? If not, we could have an assertion here instead.